.github/workflows/deploy.yml

```diff
@@ -0,0 +1,25 @@
+name: deploy
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Build
+        run: |
+          npm ci
+          npm run-script build
+
+      - name: Deploy
+        uses: JamesIves/github-pages-deploy-action@releases/v3
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          BRANCH: gh-pages
+          FOLDER: _site
```

Comment:
I _think_ GitHub pages can only deploy from either the root folder or `/docs` – at least that's what the UI shows?

> <img width="712" alt="Screenshot 2022-03-14 at 09 37 53" src="https://user-images.githubusercontent.com/30665/158145633-ea114bbd-5162-4779-b0e5-ce59689a0d01.png">

Happy to switch to `gh-pages` as the branch instead of `build` though.

Reply:
I think you should be able to deploy from a `gh-pages`, unless organisation repos are different. Will give a go and see.

Reply:
Looks like it worked. 🎉

Reply:
The docs aren't clear!